src/web/components/elements/GuVideoBlockComponent.tsx

```diff
@@ -13,18 +13,18 @@ export const GuVideoBlockComponent: React.FC<{
     credit: string;
     caption?: string;
 }> = ({ html, pillar, designType, display, credit, caption }) => {
+    const hasCaption = caption != null;
```

Comment:
Not sure it'll be `null` here?

Reply:
I would maybe just do a truthy check on `caption` because in all cases (undefined, null, empty string), we don't want to add the caption, so though the descriptive var is useful, it's probably a bit of indirection that isn't required

Reply:
Updated to adopt that